Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/entrypoints/torchserve.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+torchserve --start --model-store model_store
+
+mlflow deployments create \
+        -t torchserve \
+        -m models:${MODEL_NAME} \
+        --name news_classification \
+        -C "MODEL_FILE=news_classifier.py" \
+        -C "HANDLER=news_classifier_handler.py"\
+        -C "EXPORT_PATH=/opt/mlflow/model_store"
+
+sleep infinity
```

Comment:
This seemed like a nice solution, would it result in better logs I wonder? https://github.com/michelav/engineering.labs/blob/mav/torchserve-docker/Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/entrypoints/torchserve.sh